third_party/java/arcs/flags/flags.bzl

```diff
@@ -71,6 +71,15 @@ ARCS_BUILD_FLAGS = [
             "direct.?store?.muxer.?lru?.ttl",
         ],
     ),
+    arcs_build_flag(
+        name = "nullable_value_support",
+        desc = "Support returning nullable values",
+        bug_id = "b/174115805",
+        status = "NOT_READY",
+        stopwords = [
+            "nullable.?value.?support",
```

Comment:
Potentially worth adding `nullableof` as another stopword. If it triggers with false positives then maybe don't bother, but it would be nice to be certain that the `NullableOf` class doesn't make it to prod before it's ready.

Reply:
Added! Would be nice to be able to test in Github but I think I need the import to run the tests